attestation-service/verifier/src/cca/mod.rs

```diff
@@ -130,156 +154,104 @@ impl Verifier for CCA {
             warn!("CCA currently does not support parse `cca_realm_personalization_value`. Init data hash check skipped.");
```

Comment:
Can we support to verify this item after this PR?

Reply:
yes, I will push one additional commit to fix it.

Reply:
pls see: https://github.com/confidential-containers/kbs/pull/241/commits/90fb4b61e3e69c0cbbf993832808c080676b9da9